src/garage/experiment/local_tf_runner.py

```diff
@@ -5,11 +5,17 @@
 """
 from dowel import logger
 import psutil
-import tensorflow as tf
 
 from garage.experiment import LocalRunner
 from garage.sampler import DefaultWorker
-from garage.tf.samplers import TFWorkerClassWrapper
+
+tf = False
+TFWorkerClassWrapper = False
+try:
+    import tensorflow as tf
```

Comment:
To be honest, I'm really not a fan of this approach, since it means that tf still gets imported if it can, even if I don't want it to be imported (i.e. I have tf installed, but only want to use `garage.torch`).

It'll do for now though.

Reply:
it is really hard to do this while keeping the static analyzers happy and not accidentally creating huge messes in the import graph (I tried).

My impression of the 90% use case here is that users who aren't interested in TF won't install it (and eventually there will be `garage[tf]` and `garage[torch]`, which is what this PR is working towards).